test/utils/typesByContext.js

```diff
@@ -45,13 +53,32 @@ const createDeclarationsFilesByConfigPath = (
 const createDynamicPageTypings = testPath => {
   handlePageElementsMap(testPath, true);
   handleWidget(testPath, true);
+  handleDependencies(testPath, true);
 };
 
 const getDynamicPageTypings = testTmpDirPath => ({
   elementsMap: handlePageElementsMap(testTmpDirPath),
-  widgets: handleWidget(testTmpDirPath)
+  widgets: handleWidget(testTmpDirPath),
+  dependencies: handleDependencies(testTmpDirPath)
 });
 
+const handleDependencies = (testPath, shouldWrite = false) => {
```

Comment:
these functions (all the handle functions) do 2 things:
1. read the test example path, according to some structure we've defined for the tests
2. use corvid-types API to and save the result to some new file

so for example, the test will look like this:

const context = examplesParser.parse(testPath)
const declarations = corvidTypes.declarations(context)
expect(declarations)......



i think that:
#1 should be part of test utils (stay here)
#2 should be moved to the test itself

Reply:
I did some refactoring in order to make the test setup flow more readable